docs/_docs/features/debugger.md

```diff
@@ -32,148 +32,147 @@ project will be used to help illustrate the points described here.
 Specific details are provided for each [language](#language-specific-debugging) or
 [platform](#platform-specific-debugging), but, in general, to begin debugging code in Nuclide, you
 need to either launch a debug process from within Nuclide (e.g., iOS from the Buck toolbar) or
-attach to a  currently running process (e.g., `node`) for debugging.
+attach to a currently running process (e.g., `node`) for debugging.
 
-*Example*
+***Example***
 
-If you have a Node project running (e.g, via `npm start` or `node yourfile.js`), first press
-`cmd-shift-Y` (`ctrl-shift-Y` on Linux) to toggle the debugger. Then, attach Nuclide to the
-relevant `node` process. The number in `()` is the process id (pid).
+If you have a Node project running (e.g, via `npm start` or `node yourfile.js`), press
+`Cmd-Shift-Y` (`Ctrl-Shift-Y` on Linux) to toggle the Debugger. Then, attach Nuclide to the
+relevant `node` process. The number in "()" is the Process ID (PID).
 
 ![](/static/images/docs/feature-debugger-basics-target-process.png)
 
 > If you have multiple processes running with the same name, you can use something similar to
-> `ps aux | grep <process-name>` or Apple's Activity Monitor to find the process ID (pid) that
+> `ps aux | grep <process-name>` or Apple's Activity Monitor to find the PID that
 > matches with the process in the process list in Nuclide.
 
-> If you stop your process and restart it again, make sure you press the `Refresh List` button since
-> the pid will have changed.
+> If you stop your process and restart it again, make sure you click the **Refresh List** button since
+> the PID will have changed.
 
-Once you attach to a process, you may see a confirmation of the attachment on the command-line for
-that process.
+Once you attach to a process, you may see a confirmation of the attachment in the command-line for
+that process. <!--[MAY OR WILL?]-->
```

Comment:
At the time this was written, it wasn't always the case you would see a confirmation. However, I think theoretically, it should be "will". I won't block landing this because of that, but that can be changed in another pull request.